annotation/src/main/java/org/commonjava/maven/ext/annotation/ConfigValueProcessor.java

```diff
@@ -141,12 +141,12 @@ void generateCode() throws IOException
                 .append( value )
```

Comment:
Did you want to deal with the \n elsewhere?

Reply:
I am not sure what you mean. This is supposed to make the generated `target/property-index.md` file always have UNIX newlines, so that no matter what OS you run the build on you get the same exact file.

Reply:
Not to mention that there is already a line with hard-coded `\n`:

```
propertyIndex.append( "---\ntitle: \"Index of Properties\"\n---\n\n" );
````

which was creating lines with both `\n` and `\r\n` on Windows. Now, there is just `\n` in the file.

Reply:
OK cool